documentation/airport-guides/lrop.md

```diff
@@ -0,0 +1,36 @@
+### About this facility
+Henri Coanda International Aiport (LROP / OTP), Otopeni, Bucharest, Romania, EU
+116,718 annual operations in 2017 ([WIKIPEDIA](https://en.wikipedia.org/wiki/Henri_Coand%C4%83_International_Airport#Passengers))
```

Comment:
Cite the actual source: http://www.bucharestairports.ro/files/pages_files/AIHCB_WEBSITE__2016_-_2017__ENGL.pdf